CONTRIBUTING.md

```diff
@@ -112,17 +118,38 @@ The .strings files should still be in the original English – that’s expecte
 
 ## Adding tests
 
+### Supported devices:
+- iPhone 11 Pro Max, iOS 13.7
+- iPhone 12 Pro Max, iOS 14
```

Comment:
```suggestion
- iPhone 12 Pro Max, iOS 14.4
```

Reply:
Fixed in 1e1b41e320244d620ded7c1b8e81d03a4923a8e4 on the main branch – sorry for missing this feedback earlier.